INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 142-808-081
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:02:25] "GET / HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:02:51] "GET /index.html HTTP/1.1" 404 -
INFO _internal.py:122  * Detected change in '/home/ubantu/myproject/kesheer/kesheer/kesheer/__init__.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 142-808-081
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:32] "GET / HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:32] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:32] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:33] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:33] "GET /static/plugins/swiper/css/swiper.min.css HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:33] "GET /static/plugins/bootstrap-datepicker/css/bootstrap-datepicker.min.css HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:33] "GET /static/css/i_home/main.css HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:33] "GET /static/css/i_home/index.css HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:33] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:33] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:33] "GET /static/plugins/swiper/js/swiper.jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:33] "GET /static/plugins/bootstrap-datepicker/js/bootstrap-datepicker.min.js HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:33] "GET /static/plugins/bootstrap-datepicker/locales/bootstrap-datepicker.zh-CN.min.js HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:33] "GET /static/js/template.js HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:33] "GET /static/js/i_home/index.js HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:33] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:33] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:33] "GET /api/v1.0/session HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:33] "GET /api/v1.0/houses/index HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:33] "GET /api/v1.0/areas HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:47] "GET /register HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:50] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:51] "GET /static/css/i_home/register.css HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:51] "GET /static/js/i_home/register.js HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 14:04:51] "GET /api/v1.0/image_codes/d0a96ec9-0bb8-4b79-9117-534689a7d975 HTTP/1.1" 404 -
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 142-808-081
INFO _internal.py:122  * Detected change in '/home/ubantu/myproject/kesheer/kesheer/kesheer/api_1_0/__init__.py', reloading
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 142-808-081
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 15:10:20] "GET /api/v1.0/verify_code/sdsd HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 15:10:32] "GET /api/v1.0/image_code/sdsd HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 15:10:39] "GET /api/v1.0/image_codes/sdsd HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 15:10:43] "GET /api/v1.0/image_codes/sdsd HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 15:10:44] "GET /api/v1.0/image_codes/sdsd HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 15:10:45] "GET /api/v1.0/image_codes/sdsd HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 15:10:45] "GET /api/v1.0/image_codes/sdsd HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 15:10:46] "GET /api/v1.0/image_codes/sdsd HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 15:32:40] "GET /api/v1.0/image_codes/3456 HTTP/1.1" 200 -
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 142-808-081
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 15:57:05] "GET /api/v1.0/reister HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 15:57:09] "GET /api/v1.0/reister.html HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 15:57:44] "GET /reister.html HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 15:57:54] "GET /api/v1.0/image_codes/e53c8aef-ab63-44eb-911c-42f9b32ad360 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 15:57:57] "GET /api/v1.0/image_codes/5850b83a-8125-4349-8db0-e9d9c538cbc9 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 15:57:59] "GET /api/v1.0/image_codes/2c900a27-20f3-4cc8-8829-2df6b8028676 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 15:58:00] "GET /api/v1.0/image_codes/ec84f7dc-e0ad-4e74-9bfa-429fe0794208 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 15:58:02] "GET /api/v1.0/image_codes/4afc6567-7318-4cd0-8698-5037543bfbbc HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 15:58:03] "GET /api/v1.0/image_codes/877f9c4f-bd9d-4ceb-8192-9428adbc586c HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 15:58:04] "GET /api/v1.0/image_codes/973e7163-53cd-4f3f-9044-71c16e21f920 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [28/Apr/2019 15:58:09] "GET /api/v1.0/image_codes/f26b1ec8-25d1-45d6-a57d-5f5ee73ede8c HTTP/1.1" 200 -
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 142-808-081
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 142-808-081
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122  * Restarting with stat
WARNING _internal.py:122  * Debugger is active!
INFO _internal.py:122  * Debugger PIN: 142-808-081
INFO _internal.py:122  * Detected change in '/home/ubantu/myproject/kesheer/kesheer/config.py', reloading
INFO _internal.py:122  * Restarting with stat
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
ERROR app.py:1761 Exception on /register.html [GET]
Traceback (most recent call last):
  File "/home/ubantu/.virtualenvs/ihome/local/lib/python2.7/site-packages/flask/app.py", line 2291, in wsgi_app
    ctx.push()
  File "/home/ubantu/.virtualenvs/ihome/local/lib/python2.7/site-packages/flask/ctx.py", line 377, in push
    self.app, self.request
  File "/home/ubantu/.virtualenvs/ihome/local/lib/python2.7/site-packages/flask_session/sessions.py", line 132, in open_session
    val = self.redis.get(self.key_prefix + sid)
  File "/home/ubantu/.virtualenvs/ihome/local/lib/python2.7/site-packages/redis/client.py", line 1264, in get
    return self.execute_command('GET', name)
  File "/home/ubantu/.virtualenvs/ihome/local/lib/python2.7/site-packages/redis/client.py", line 772, in execute_command
    connection = pool.get_connection(command_name, **options)
  File "/home/ubantu/.virtualenvs/ihome/local/lib/python2.7/site-packages/redis/connection.py", line 994, in get_connection
    connection.connect()
  File "/home/ubantu/.virtualenvs/ihome/local/lib/python2.7/site-packages/redis/connection.py", line 497, in connect
    raise ConnectionError(self._error_message(e))
ConnectionError: Error 111 connecting to 127.0.0.1:6379. Connection refused.
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:10:15] "GET /register.html HTTP/1.1" 500 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:10:36] "GET /register.html HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:10:36] "GET /static/plugins/bootstrap/css/bootstrap.min.css HTTP/1.1" 304 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:10:36] "GET /static/plugins/font-awesome/css/font-awesome.min.css HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:10:36] "GET /static/css/reset.css HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:10:36] "GET /static/css/i_home/main.css HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:10:36] "GET /static/css/i_home/register.css HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:10:36] "GET /static/js/jquery.min.js HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:10:36] "GET /static/plugins/bootstrap/js/bootstrap.min.js HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:10:36] "GET /static/js/i_home/register.js HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:10:36] "GET /static/images/logo@128x59.png HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:10:36] "GET /static/plugins/font-awesome/fonts/fontawesome-webfont.woff2?v=4.6.3 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:10:37] "GET /api/v1.0/image_codes/b136f2ae-8131-4afa-9b7b-fa1cfdf0eb46 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:10:54] "GET /api/v1.0/image_codes/86fa59e9-1189-4efa-9589-004ce1b5b5f7 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:10:55] "GET /api/v1.0/image_codes/38be064f-484a-4823-a8cf-900fc5776018 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:10:57] "GET /api/v1.0/image_codes/cd67e761-98a4-424c-b933-66c4fc5ba8f5 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:14:23] "GET /sms_codes HTTP/1.1" 404 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:15:12] "GET /api/v1.0/sms_codes/18829711180 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:17:44] "GET /api/v1.0/image_codes/489fc4ed-719d-4996-b4c2-b1047d2fe145 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:18:17] "GET /api/v1.0/sms_codes/18829711180 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:18:36] "GET /api/v1.0/sms_codes/18829711180 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:18:43] "GET /api/v1.0/sms_codes/18829711180 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:20:32] "GET /api/v1.0/image_codes/de6898c2-ce51-4863-966a-9e99144501eb HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:22:18] "GET /api/v1.0/sms_codes/18829711180 HTTP/1.1" 200 -
ERROR verify_code.py:80 'NoneType' object has no attribute 'decode'
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:23:44] "GET /api/v1.0/sms_codes/18829711180?image_code=UIRL&image_code_id=de6898c2-ce51-4863-966a-9e99144501eb HTTP/1.1" 200 -
ERROR verify_code.py:80 'NoneType' object has no attribute 'decode'
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:24:36] "GET /api/v1.0/sms_codes/18829711180?image_code=UIRL&image_code_id=de6898c2-ce51-4863-966a-9e99144501eb HTTP/1.1" 200 -
ERROR verify_code.py:80 'NoneType' object has no attribute 'decode'
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:25:03] "GET /api/v1.0/sms_codes/18829711180?image_code=UIRL&image_code_id=de6898c2-ce51-4863-966a-9e99144501eb HTTP/1.1" 200 -
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:25:15] "GET /api/v1.0/sms_codes/18829711180?image_code=UIRL&image_code_id=de6898c2-ce51-4863-966a-9e99144501eb HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:25:53] "GET /api/v1.0/image_codes/b22cb328-68d1-4c91-822f-49def070c351 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:25:58] "GET /api/v1.0/image_codes/f2cf0a71-3548-45f5-9828-aa71df04d8a3 HTTP/1.1" 200 -
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:26:43] "GET /api/v1.0/sms_codes/18829711180?image_code=HNCW&image_code_id=f2cf0a71-3548-45f5-9828-aa71df04d8a3 HTTP/1.1" 200 -
INFO _internal.py:122  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:122 127.0.0.1 - - [29/Apr/2019 10:49:49] "GET /api/v1.0/sms_codes/18829711180?image_code=HNCW&image_code_id=f2cf0a71-3548-45f5-9828-aa71df04d8a3 HTTP/1.1" 200 -
